Tabloid/client/src/modules/TagView.js

```diff
@@ -0,0 +1,32 @@
+import { getAllTags } from './TagManagment';
```

Comment:
This is a component module, so it should not be in the modules directory. Please move this to the components directory.

Reply:
Do I need to move TagManagement to modules or TagView ?